code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -194,7 +194,21 @@
 			new /obj/item/dnainjector/cryokinesis(src)
 			new /obj/item/gun/energy/temperature/security(src)
 			new /obj/item/melee/transforming/energy/sword/saber/blue(src) //see see it fits the theme bc its blue and ice is blue
-
+			
+			if("neo")
```

Comment:
```suggestion
		if("neo")
```

Reply:
If statemwnt was tabbed too many times that's what's causing it to not work

Reply:
thanks theos